packages/ember-cli-fastboot/package.json

```diff
@@ -49,57 +49,57 @@
     "silent-error": "^1.1.1"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.21.3",
-    "@babel/plugin-proposal-decorators": "^7.21.0",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.1.1",
-    "@ember/test-helpers": "^2.9.3",
+    "@babel/eslint-parser": "^7.24.1",
+    "@babel/plugin-proposal-decorators": "^7.24.1",
+    "@ember/optional-features": "^2.1.0",
+    "@ember/test-helpers": "^3.3.0",
     "@embroider/test-setup": "^3.0.3",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "broccoli-asset-rev": "^3.0.0",
     "broccoli-test-helper": "^1.5.0",
-    "concurrently": "^8.0.1",
-    "ember-auto-import": "^2.6.3",
-    "ember-cli": "~4.12.3",
-    "ember-cli-dependency-checker": "^3.3.1",
-    "ember-cli-htmlbars": "^6.2.0",
+    "concurrently": "^8.2.2",
+    "ember-auto-import": "^2.7.2",
+    "ember-cli": "~5.8.1",
+    "ember-cli-clean-css": "^3.0.0",
+    "ember-cli-dependency-checker": "^3.3.2",
+    "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^7.0.0",
-    "ember-qunit": "^6.2.0",
-    "ember-resolver": "^10.0.0",
-    "ember-source": "~4.12.0",
+    "ember-page-title": "^8.2.3",
+    "ember-qunit": "^8.0.2",
+    "ember-resolver": "^11.0.1",
+    "ember-source": "~5.8.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^5.7.2",
+    "ember-template-lint": "^5.13.0",
     "ember-try": "^3.0.0",
-    "eslint": "^8.37.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-ember": "^11.5.0",
-    "eslint-plugin-n": "^15.7.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-qunit": "^7.3.4",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^11.12.0",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-qunit": "^8.1.1",
     "glob": "^7.1.3",
     "lint-to-the-future": "^2.0.0",
     "lint-to-the-future-eslint": "^2.0.1",
     "loader.js": "^4.7.0",
     "mocha": "^9.1.2",
-    "prettier": "^2.8.7",
-    "qunit": "^2.19.4",
+    "prettier": "^3.2.5",
+    "qunit": "^2.20.1",
     "qunit-dom": "^2.0.0",
     "sinon": "^17.0.1",
-    "stylelint": "^15.4.0",
-    "stylelint-config-standard": "^32.0.0",
-    "stylelint-prettier": "^3.0.0",
-    "webpack": "^5.90.1"
+    "stylelint": "^15.11.0",
+    "stylelint-config-standard": "^34.0.0",
+    "stylelint-prettier": "^4.1.0",
+    "webpack": "^5.91.0"
   },
   "peerDependencies": {
     "ember-source": "^4.0.0"
   },
   "engines": {
-    "node": "14.* || 16.* || >= 18"
```

Comment:
just FYI looks like it would be a breaking change

Reply:
thanks! that wasn't intentional, so I reverted it 👍